### PR TITLE
Pitch one sound instead of using multiple ones

### PR DESCRIPTION
**Pull request by tacotexmex**

### CODE REVIEW

File: init.lua [hunk 1]
```diff
@@ -1,4 +1,5 @@
 local load_time_start = minetest.get_us_time()
+math.randomseed(os.time())
```
Comment (HybridDog) [inline init.lua:2]: You don't need to use randomseed I think.

Comment (tacotexmex) [reply init.lua:2]: I think it's needed. Not using it made the pitch variation predictable. You can probably hear it too if you test it.

Comment (HybridDog) [reply init.lua:2]: The function is executed in builtin: https://github.com/minetest/minetest/blob/0c319469439237ba0db31ccbffeef06e2f5ab45f/builtin/init.lua#L23

Comment (tacotexmex) [reply init.lua:2]: Hmm, need to look at it again to make sure.

Comment (SmallJoker) [reply init.lua:2]: @tacotexmex

**Comment by tacotexmex** [comment]
I've decided to close the PR, since it assumes that a game maker wants the same pickup sound, only pitched differently. It's better to have three different sound files for when one wants to use completely different sounds randomly.

**Comment by HybridDog** [comment]
You could do both: multiple sounds and a slightly random pitch

**Comment by tacotexmex** [comment]
True, true. We’ll see.

**Pull request closed without merging**